iac-common/src/test/java/org/sonar/iac/common/yaml/object/BlockObjectTest.java

```diff
@@ -93,6 +94,19 @@ void attribute() {
     assertThat(absentAttr.status).isEqualTo(YamlObject.Status.ABSENT);
   }
 
+  @Test
+  void list() {
+    BlockObject block = BlockObject.fromPresent(ctx, parseMap("foo: [bar, car]"), "a");
+    ListObject listPresent = block.list("foo");
+    assertThat(listPresent.status).isEqualTo(YamlObject.Status.PRESENT);
+    assertThat(listPresent.getItemIf(e -> true)
```

Comment:
`getItemIf` is only used for testing right now. Maybe you can make the field visible for testing by making it package protected so you can access the items without the predicate.